docs/framework/unmanaged-api/profiling/icorprofilerinfo13-createhandle-method.md

```diff
@@ -0,0 +1,60 @@
+---
+description: "Learn more about: ICorProfilerInfo13::CreateHandle Method"
+title: "ICorProfilerInfo13::CreateHandle Method"
+ms.date: "03/19/2021"
+api_name: 
+  - "ICorProfilerInfo13.CreateHandle"
+api_location: 
+  - "coreclr.dll"
+  - "corprof.idl"
+api_type: 
+  - "COM"
+---
+# ICorProfilerInfo13::CreateHandle Method
+
+Creates a handle that wraps a specified object.
+  
+## Syntax  
+  
+```cpp  
+    HRESULT CreateHandle(
+                [in] ObjectID            object,
+                [in] COR_PRF_HANDLE_TYPE type,
+                [out] ObjectHandleID*    pHandle);
+```  
+  
+## Parameters
+
+`object`
+[in] The object reference to wrap with a handle.
+
+`type`
+[in] The type of handle to create.
+
+`pHandle`
+[out] A caller-provided pointer that will point to the handle created to wrap the specified `object`.
+
+## Remarks
+
+Following are the expected usages depending on the specified `COR_PRF_HANDLE_TYPE` value:
+ - COR_PRF_HANDLE_TYPE_WEAK: Monitors if an object stays in memory over time. If the wrapped object has been collected, [ICorProfilerInfo13::GetObjectIDFromHandle](icorprofilerinfo13-getobjectidfromhandle-method.md) will return a null `ObjectID` for its wrapping weak handle.
+- `COR_PRF_HANDLE_TYPE_STRONG`: Enforces that an object survives garbage collections even though no other object references it.
```

Comment:
Somehow it looks like my suggestions to fix these warnings were lost.

Reply:
> Somehow it looks like my suggestions to fix these warnings were lost.

@gewarren: Let me know if the last push fixed the indentation issue